src/store/dialog/dialog-reducer.ts

```diff
@@ -6,9 +6,9 @@ import { DialogAction, dialogActions } from "./dialog-actions";
 
 export type DialogState = Record<string, Dialog>;
 
-export interface Dialog {
+export interface Dialog <T = any> {
```

Comment:
it's better to use `{}` instead of `any`. This way the consumer of this interface would have to provide a type if he wanted sth to be stored in `data` field